.eleventy.js

```diff
@@ -12,7 +13,24 @@ module.exports = function (eleventyConfig) {
   eleventyConfig.addPlugin(embedYouTube);
   eleventyConfig.addPassthroughCopy({ _src: '/' });
 
+  // Allow us to put rendered markdown in HTML
   eleventyConfig.addPairedShortcode('markdown', (content) => {
     return rdmd.html(content);
   });
+
+  // Minify our build to save the bits
+  eleventyConfig.addTransform('htmlmin', function (content, outputPath) {
+    if (outputPath?.endsWith('.html')) {
```

Comment:
This line throwing an error locally:

```shell
Eleventy fatal error: (more in DEBUG output)
> Error in your Eleventy config file '/Users/alexroan/Documents/Projects/UIs/documentation/.eleventy.js'.

`EleventyConfigError` was thrown
> Unexpected token '.'

`SyntaxError` was thrown:
    /Users/alexroan/Documents/Projects/UIs/documentation/.eleventy.js:23
        if (outputPath?.endsWith('.html')) {
                       ^
    
    SyntaxError: Unexpected token '.'
        at wrapSafe (internal/modules/cjs/loader.js:915:16)
        at Module._compile (internal/modules/cjs/loader.js:963:27)
        at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
        at Module.load (internal/modules/cjs/loader.js:863:32)
        at Function.Module._load (internal/modules/cjs/loader.js:708:14)
        at Module.require (internal/modules/cjs/loader.js:887:19)
        at require (internal/modules/cjs/helpers.js:74:18)
        at TemplateConfig.mergeConfig (/Users/alexroan/Documents/Projects/UIs/documentation/node_modules/@11ty/eleventy/src/TemplateConfig.js:87:23)
        at new TemplateConfig (/Users/alexroan/Documents/Projects/UIs/documentation/node_modules/@11ty/eleventy/src/TemplateConfig.js:25:24)
        at Object.<anonymous> (/Users/alexroan/Documents/Projects/UIs/documentation/node_modules/@11ty/eleventy/src/Config.js:5:14)
error Command failed with exit code 1.
```

Reply:
Looks like we need to specify a version of node so we are all on the same version. What version are you on?

Reply:
This is a newer is feature called optional chaining. much cleaner than the older

If a and a.b and a.b.c and a.b.c.d=...

Reply:
I double checked, you need at least node 14 (which is current https://nodejs.org/en/about/releases/). I've added this as a hard requirement in #67